code401challenges/src/main/java/code401challenges/BinarySearch.java

```diff
@@ -0,0 +1,37 @@
+package code401challenges;
+
+import java.util.Arrays;
+
+public class BinarySearch {
+    public static int BinaryMachine(int[] startArray, int pinpointNum) {

```

Comment:
Nitpick: We name all methods with lowerCamelCase, not UpperCamelCase; we save UpperCamelCase for class names.